ynr/apps/candidates/views/mixins.py

```diff
@@ -21,9 +21,9 @@ def get_leaderboards(self, all_time=True):
                 timezone.now(),
             ),
             (
-                _("2018 local elections"),
-                timezone.make_aware(parse("2018-03-01")),
-                timezone.make_aware(parse("2018-05-03")),
+                _("2019 local elections"),
+                timezone.make_aware(parse("2019-03-01")),
+                timezone.make_aware(parse("2019-05-10")),
```

Comment:
What's the significance of the 10th May here?
Are you thinking about including results entry?

Reply:
Yeah, a random date where I assume not many edits will be done for future elections. I guess that might change if we have another major poll, but this is just for the display rather than something that alters the data in any way.